### ISSUE TITLE
Credit Note building in XML

### ISSUE THREAD

**Issue by vanjoechua** [klsheng/myinvois-php-sdk#12]
Tried to create a CN using the example and had errors when adding the Billing references. Using the LHDN 1.1 example of a CN and comments from the FB MyInvois group it seems the Billing Reference cannot have an Additional Document Reference Node. I had to remove some code from UBL\BillingReference to disable adding the additional document reference. Hope this can help.

 ```
    public function validate()
    {
//        if ($this->additionalDocumentReference === null) {
//            throw new InvalidArgumentException('Missing additionalDocumentReference');
//        }
    }

   public function xmlSerialize(Writer $writer): void
    {
        $this->validate();

//        $writer->write([
//            XmlSchema::CAC . 'AdditionalDocumentReference' => $this->additionalDocumentReference
//        ]);

        if ($this->invoiceDocumentReference !== null) {
            $writer->write([
                XmlSchema::CAC . 'InvoiceDocumentReference' => $this->invoiceDocumentReference
            ]);
        }
    }
```

**Comment by klsheng** [comment]
Thanks for the info. The fix is in 1.0.3.

**Comment by vanjoechua** [comment]
You are welcome!
